projects/common/src/telemetry/user-telemetry-impl.service.ts

```diff
@@ -71,6 +71,21 @@ export class UserTelemetryImplService extends UserTelemetryService {
       );
   }
 
+  private getDifferenceInDays(timeParamValue: string | undefined): number | undefined {
+    if (timeParamValue !== undefined) {
+      const timeRangeArray = timeParamValue.split('-');
```

Comment:
I think we can leverage the `getDate` method and get the difference in days.

```suggestion
      const timeRangeArray = /(\d+)-(\d+)/.exec(timeRangeValue);
      const todayDate = new Date();
      const searchStartDate = new Date(+timeRangeArray[1]);
      return todayDate.getDate() - searchStartDate.getDate();
```